src/js/pages/Home.js

```diff
@@ -1,8 +1,17 @@
 import React, { Component } from 'react';
+import { get } from 'lodash';
+
+import SecondaryContentBanner from 'js/page_sections/SecondaryContentBanner';
+import Service311 from 'js/page_sections/Service311';
 import GlobalSearch from 'js/modules/GlobalSearch';
 import Hero from 'js/modules/Hero';
+import ExternalLink from 'js/modules/ExternalLink';
 import homepageImg from 'images/lady_bird_lake.jpg';
 
+import jsonFileData from '__tmpdata/services';
+const services311 = get(jsonFileData, "services311", null);
+
+
 class Home extends Component {
```

Comment:
Our other pages are functional. It seems like the homepage doesn't require state, so could we make this consistent with other pages that are stateless components.

Reply:
yup, I was working on this when you jumped in to review. great minds...